Feature/feature group key view

mackenziekilgore opened the pull request.

Commit:
LIBITD-2532. Created view for the groups/group_key route

Created a class for the group_key view and added URL configuration for it

https://umd-dit.atlassian.net/browse/LIBITD-2532

Commit:
LIBITD-2535. Fixed style issues

Deleted the underscore in the class name and deleted some unnecessary spaces.

https://umd-dit.atlassian.net/browse/LIBITD-2535

peichman-umd commented on src/ipmanager/urls.py:
```diff
@@ -17,10 +17,11 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('groups/', GroupsView.as_view())
```
Needs a comma at the end of this line.

peichman-umd reviewed (changes requested):
Style is good, just need to add a comma to a line in the `urls.py` file.

Commit:
LIBITD-2535. Added the commas in urlpatterns

Commas needed between each path and after so I added those

https://umd-dit.atlassian.net/browse/LIBITD-2535

peichman-umd reviewed (approved):
Looks good!
